package.json

```diff
@@ -8,8 +8,8 @@
     "get-all-submodules": "git submodule update --init --remote",
     "compile-posdao-contracts": "cd ./posdao-contracts && npm i && npm run compile",
     "make-spec": ". ./scripts/network-spec && cd ./posdao-contracts && npm i && node ./scripts/make_spec.js",
-    "start-test-setup": "bash scripts/start-test-setup ../parity-ethereum/target/debug/parity",
-    "test": "./node_modules/.bin/truffle migrate --network development && ./node_modules/.bin/truffle test --network development && npm run watcher",
+    "start-test-setup": "bash scripts/start-test-setup ../parity-ethereum/target/debug/parity && bash scripts/deploy-staking-token.sh",
```

Comment:
Let's move `bash scripts/deploy-staking-token.sh` to `npm run test`.

Reply:
Wouldn't it make more sense to keep it here, because without a staking token (I mean erc20) deployed and set in ValidatorSetAuRa, the network is kind of not "started" properly?

Reply:
I mean, is there a scenario where you would want to launch parity nodes, but not deploy staking token contract?

Reply:
Sometimes I just used `npm run start-test-setup` to see how the basic part of the network worked (without deploying the staking token it would start a bit faster). Ok, let's leave it as it is, no problem.